modules/ROOT/pages/afm-monitoring-private-apis.adoc

```diff
@@ -0,0 +1,53 @@
+= Monitoring the Endpoints of Private APIs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+Private APIs are APIs with endpoints that are accessible only within your network. API Functional Monitoring lets you monitor private APIs if you have a subscription to Anypoint Virtual Private Cloud (VPC). With VPC, you can run workers in CloudHub in a virtual, private, and isolated network segment, rather than in a region in which resources are shared.
+
+A _private location_ is a worker that runs in an instance of VPC that is configured to allow workers to access your private network. AFM can create workers within VPC that can run tests against APIs that are accessible only within the network that VPC is configured for. The APIs must be deployed in CloudHub. You can create multiple private locations in a single VPC.
+
+The tests in monitors are run on schedules. For a private location to run according to a schedule, a schedule must be associated with it. A private location can be associated with more than one schedule for more than one monitor.
+
+Before you can run tests from a private location, you must create one, which involves specifying a name and an environment for the worker to run it. The environment must be associated with a VPC and must be the same environment that an API to be tested is deployed in. For example, you might create a private location that you name `myLocation` and set it to run in environment `myEnvironment`. When you choose to run a test in `myLocation`, AFM allocates a worker in `myEnvironment` in Anypoint Virtual Private Cloud.
```

Comment:
"you must create one": does "one" refer to "tests" or "location"?

Reply:
"...specifying...an environment for the worker to run it" : I want to add "in" somewhere here, "in" an environment.

Reply:
Fixed.